services/net/content/ContentManager.cs

```diff
@@ -106,17 +129,28 @@ private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> resul
             Transcription = "",
             SourceUrl = result.Message.Value.Link,
             PublishedOn = result.Message.Value.PublishedOn,
-            Actions = Array.Empty<ContentActionModel>(),
-            Categories = Array.Empty<ContentCategoryModel>(),
-            Tags = Array.Empty<ContentTagModel>(),
-            TonePools = Array.Empty<ContentTonePoolModel>(),
-            FileReferences = Array.Empty<FileReferenceModel>()
         };
 
         // Only add if doesn't already exist.
         var exists = await _api.FindContentByUidAsync(content.Uid, content.Source);
         if (exists == null)
-            await _api.AddContentAsync(content);
+        {
+            content = await _api.AddContentAsync(content);
+
+            // Upload the file to the API.
+            if (content != null && !String.IsNullOrWhiteSpace(result.Message.Value.FilePath))
+            {
+                // TODO: Handle different storage locations.
+                // Remote storage locations may not be easily accessible by this service.
+                var sourcePath = Path.Join(_options.ClipPath, result.Message.Value.FilePath);
+                if (File.Exists(sourcePath))
+                {
+                    var file = File.OpenRead(sourcePath);
+                    var fileName = Path.GetFileName(sourcePath);
+                    await _api.UploadFileAsync(content.Id, content.Version ?? 0, file, fileName);
```

Comment:
I don't love this solution as a large file will block the service for a while.  However, it is the simplest way to support remote architectural complexity issues.